src/scss/packages/usa-image-example/src/_styles.scss

```diff
@@ -0,0 +1,43 @@
+@use 'uswds-core' as *;
+
+.usa-image-example {
+  figure {
+    margin: 0;
+    display: table;
+  }
+
+  figcaption {
+    margin-top: units(1.5);
+    display: table-caption;
+    caption-side: bottom;
+  }
+
+  .img--bordered {
+    border-top: units(1px) solid color('base-lighter');
+    border-right: units(1px) solid color('base-lighter');
+    border-left: units(1px) solid color('base-lighter');
+  }
+}
+
+@mixin usa-image-example($color) {
+  img {
+    box-shadow: 0 units(1) 0 0 color($color);
```

Comment:
This was `border-bottom` but the combination of a thick `border-bottom` and a thin border on all the other sides made the `border-bottom` have tapered sides. This changes all the edges to be 90degrees.